From b2a79a6f2933e969b2b70d466e03741d7ecc0ad8 Mon Sep 17 00:00:00 2001
From: "Sodhi, VunnyX" <vunnyx.sodhi@intel.com>
Date: Wed, 11 Mar 2015 16:58:57 +0530
Subject: [PATCH] ASoC: Intel: Changing to codec dapm instead of sound card
 dapm

Changing to codec's dapm instead of sound card's dapm
to remove unknown pin error messages during boot.

Issue: GMINL-6990

Change-Id: I5e2171be7a78fe64cd5b0930ab83261d4c36dae7
Signed-off-by: Sodhi, VunnyX <vunnyx.sodhi@intel.com>
---
 sound/soc/intel/board/cht_bl_dpcm_rt5672.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sound/soc/intel/board/cht_bl_dpcm_rt5672.c b/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
index df5a6cb..e1109cf 100644
--- a/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
+++ b/sound/soc/intel/board/cht_bl_dpcm_rt5672.c
@@ -699,11 +699,11 @@ static int cht_audio_init(struct snd_soc_pcm_runtime *runtime)
 	 * suspend. Mark the end points ignore_suspend
 	 */
 	/*TODO: CHECK this */
-	snd_soc_dapm_ignore_suspend(&card->dapm, "HPOL");
-	snd_soc_dapm_ignore_suspend(&card->dapm, "HPOR");
+	snd_soc_dapm_ignore_suspend(&codec->dapm, "HPOL");
+	snd_soc_dapm_ignore_suspend(&codec->dapm, "HPOR");
 
-	snd_soc_dapm_ignore_suspend(&card->dapm, "SPOL");
-	snd_soc_dapm_ignore_suspend(&card->dapm, "SPOR");
+	snd_soc_dapm_ignore_suspend(&codec->dapm, "SPOL");
+	snd_soc_dapm_ignore_suspend(&codec->dapm, "SPOR");
 
 	snd_soc_dapm_enable_pin(&card->dapm, "Headset Mic");
 	snd_soc_dapm_enable_pin(&card->dapm, "Headphone");
-- 
1.9.1

